architecture/adapters.md

```diff
@@ -12,16 +12,35 @@ They can be written in any language, and must simply respond to a set of HTTP en
 ## Basic Concept
 Here's a diagram illustrating the basic concept of Yacs and adapters:
 
-<!-- ![alt-text](../_media/adaptersfig1.png) -->
+<!-- ![alt-text](../_media/adaptersfig1.png) - ->
 ========  
 DIAGRAM GOES HERE  
 ========  
-<!-- TODO: Make diagram -->
-<!--
-When Yacs needs information that isn't stored in its database, it sends an HTTP request to the adapter that is responsible for that information. Then that adapter works its magic, and sends the information back in JSON format. This document is going to focus on the [`yaml-rpi`][yaml-rpi-adapter] adapter and use its code in examples.
+<!-- TODO: Make diagram --><!--
+
+- ->
+When Yacs needs information that isn't stored in its database, it sends an HTTP request to the adapter that is responsible for that information. Then that adapter works its magic, and sends the information back in JSON format. This document is going to focus on the [`yaml-rpi`][yaml-rpi-adapter] adapter and use its code in examples. The document `app.rb` will also be edited for clarity to this:
```

Comment:
This isn't technically correct. Every adapter is polled on a fixed interval, and Yacs saves the combined result/